src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -1044,8 +1044,8 @@ static MemorySegment ofArray(double[] doubleArray) {
      *}
      * <p>
      * On 32-bit platforms, the given address value will be normalized such that the
-     * returned memory segment's method {@link MemorySegment#address() address} will always return
-     * zeros for the upper 32 bits of the address.
+     * upper 32 bits of the {@link MemorySegment#address() address} of returned memory segment
```

Comment:
Sorry, I mean to replicate this change in all the other pieces of javadoc where you talk of 32 bits.

Reply:
Fixed now